randovania/games/dread/gui/dialog/game_export_dialog.py

```diff
@@ -84,6 +81,35 @@ def romfs_validation(line: QtWidgets.QLineEdit):
 
 
 class DreadGameExportDialog(GameExportDialog, Ui_DreadGameExportDialog):
+    def get_path_to_ryujinx(self) -> Path:
+        ryujinx_path_tuple = ("Ryujinx", "mods", "contents", "010093801237c000")
+        current_system = platform.system()
+        if current_system == "Windows":
+            return Path(os.environ["APPDATA"], *ryujinx_path_tuple)
+        elif current_system == "Linux":
+            if self.linux_ryujinx_path == LinuxRyujinxPath.NATIVE:
+                base_config_path = QtCore.QStandardPaths.writableLocation(
+                    QtCore.QStandardPaths.StandardLocation.GenericConfigLocation
```

Comment:
This will need to get changed for flatpak (or we just hide native), because GenericConfigLocation will get redirected to ~/.var/app/io.github.randovania.Randovania/config